tests/IteratorTest.php

```diff
@@ -99,8 +99,9 @@ public function testNext()
 
     public function testRewind()
     {
+        self::$iterator->next();
```

Comment:
Can you explain the change behind this test case?

Reply:
in your initial test case iterator may already be ate the first position. So, this change will make sure, you are running rewinding function on array being iterated.

Reply:
Ok that makes sense